src/constants/routes.ts

```diff
@@ -0,0 +1,5 @@
+export const ROUTE_HOME = '/';
+export const ROUTE_LOGIN = '/login';
+export const ROUTE_MY = '/my';
+export const ROUTE_ORDER = '/order';
+export const ROUTE_NOT_FOUND = '*';
```

Comment:
네임스페이스로 묶어서 다른 파일에서 사용할 때 자동완성을 편하게 하는 방법도 있을 것 같아요!
```suggestion
export const ROUTE = {
  HOME: '/',
  ...
} as const
```